### ISSUE TITLE
TypeError: Cannot promisify an API that has normal methods with 'Async'-suffix

### ISSUE THREAD

**Issue by canercandan** [jdxcode/npm-register#135]
```
$ node --version
v8.12.0
```
```
$ npm-register --version
2.7.1
```

I am getting this error when I start the `npm-register` server with S3 set.

```
Opbeat agent is inactive due to configuration
/usr/local/share/.config/yarn/global/node_modules/bluebird/js/release/promisify.js:54
                    throw new TypeError("Cannot promisify an API that has normal methods with '%s'-suffix\u000a\u000a    See http://goo.gl/MqrFmX\u000a"
                    ^

TypeError: Cannot promisify an API that has normal methods with 'Async'-suffix

    See http://goo.gl/MqrFmX
```

**Comment by canercandan** [comment]
@dgautsch for some reasons it seems that the NPM package does not contain the latest version tagged on the git repo `2.7.1 <> 2.8.1` I will check if the last version fix the issue

**Comment by canercandan** [comment]
Yep I confirm this issue does not exist with the latest version

**Comment by dgautsch** [comment]
@canercandan yea i haven't published 2.8.1 yet because I wanted to do some more testing. Thanks for checking!

**Comment by canercandan** [comment]
Thanks @dgautsch for the heads-up though the lastest npm package version available is not working with S3 which is quite sad, which version shall we use (available in NPM) to get S3 working with? thanks

**Comment by dgautsch** [comment]
@canercandan I've published 2.8.1, please give it a try. Thanks!
